packages/SystemUI/res-keyguard/values-it/aosip_strings.xml

```diff
@@ -0,0 +1,141 @@
+        <item>Otto</item>
+        <item>Nove</item>
+        <item>Dieci</item>
+        <item>Undici</item>
+        <item>Dodici</item>
+        <item>Tredici</item>
+        <item>Quattordici</item>
+        <item>Quindici</item>
+        <item>Sedici</item>
+        <item>Diciassette</item>
+        <item>Diciotto</item>
+        <item>Diciannove</item>
+        <item>Venti</item>
+        <item>Ventuno</item>
+        <item>Ventitre</item>
+        <item>Ventiquattro</item>
+    </string-array>
+
+    <!-- [CHAR LIMIT=20] -->
+    <string-array name="type_clock_minutes">
+        <item> </item>
+        <item>e uno</item>
+        <item>e due</item>
+        <item>e tre</item>
+        <item>e quattro</item>
+        <item>e cinque</item>
+        <item>e sei</item>
+        <item>e sette</item>
+        <item>e otto</item>
+        <item>e nove</item>
+        <item>e dieci</item>
+        <item>e undici</item>
+        <item>e dodici</item>
+        <item>e tredici</item>
+        <item>e quattordici</item>
+        <item>e quindici</item>
+        <item>e sedici</item>
+        <item>e diciassette</item>
+        <item>e diciotto</item>
+        <item>e diciannove</item>
+        <item>e venti</item>
+        <item>e ventuno</item>
+        <item>e ventidue</item>
+        <item>e ventitre</item>
+        <item>e ventiquattro</item>
+        <item>e venticinque</item>
+        <item>e ventisei</item>
+        <item>e ventisette</item>
+        <item>e ventotto</item>
+        <item>e ventinove</item>
+        <item>e trenta</item>
+        <item>e trentuno</item>
+        <item>e trentadue</item>
+        <item>e trentatre</item>
+        <item>e trentaquattro</item>
+        <item>e trentacinque</item>
+        <item>e trentasei</item>
+        <item>e trentasette<item>
+        <item>e trentotto</item>
+        <item>e trentanove</item>
+        <item>e quaranta</item>
+        <item>e quarantuno</item>
+        <item>e quarantadue</item>
+        <item>e quarantatre</item>
+        <item>e quarantaquattro</item>
+        <item>e quarantecinque</item>
+        <item>e quarantasei</item>
+        <item>e quarantasette</item>
+        <item>e quarantotto</item>
+        <item>e quarantanove</item>
+        <item>e cinquanta</item>
+        <item>e cinquantuno</item>
+        <item>e cinquantadue</item>
+        <item>e cinquantatre</item>
+        <item>e cinquantaquattro</item>
+        <item>e cinquantacinque</item>
+        <item>e cinquantasei</item>
+        <item>e cinquantasette</item>
+        <item>e cinquantotto</item>
+        <item>e cinquantanove</item>
+    </string-array>
+
+    <!-- [CHAR LIMIT=8] -->
+    <string name="clock_title_default" translatable="false">Standard</string>
+    <!-- [CHAR LIMIT=8] -->
+    <string name="clock_title_bubble" translatable="false">Bolla</string>
+    <!-- [CHAR LIMIT=8] -->
+    <string name="clock_title_analog" translatable="false">Analogico</string>
+    <!-- [CHAR LIMIT=8] -->
```

Comment:
and you may wanna actually care about those char limits :)

Reply:
Uhm, the german translation even has more characters, that's why i didn't thought that out